ext/pdo/tests/pdo_036.phpt

```diff
@@ -15,9 +15,9 @@ var_dump($x);
 
 ?>
 --EXPECTF--
-object(PDOStatement)#%d (1) {
+object(PDOStatement)#%d (0) {
   ["queryString"]=>
-  NULL
+  uninitialized(string)
```

Comment:
Hm, I guess this shows that it needs to be nullable after all...

Reply:
`queryString` is only initialized to a string in the constructor (`pdo_stmt_construct`), so it's not done when using `ReflectionClass::newInstance()`. However, my assumption was that since it's a readonly typed property, the uninitialized default value is good enough instead of `null`.

Reply:
Oh, I missed that `ReflectionClass::newInstance()` calls the constructor :/ Then yes, we need to make it nullable